LICENSE

```diff
@@ -1,6 +1,8 @@
 Vega Strike - Space Simulation, Combat and Trading
-Copyright (C) 2001-2023 Daniel Horn, pyramid3d, Stephen G. Tuggy,
-Roy Falk, Benjamen R. Meyer, and other Vega Strike Contributors
+Copyright (C) 2001-2025 The Vega Strike Contributors:
+Project creator: Daniel Horn
```

Comment:
@BenjamenMeyer I'm hoping that adding the word `Project` in here will help improve clarity. Let me know if this works for you for most files, or not.